mkl_umath/src/mkl_umath_loops.c.src

```diff
@@ -67,40 +67,36 @@
 
 #define MKL_INT_MAX ((npy_intp) ((~((MKL_UINT) 0)) >> 1))
 
-#define CHUNKED_VML_CALL2(vml_func, n, type, in1, op1)   \
-    do {                                                 \
-        npy_intp _n_ = (n);                              \
-        const npy_intp _chunk_size = MKL_INT_MAX;        \
-        type *in1p = (type *) (in1);                     \
-        type *op1p = (type *) (op1);                     \
-        while (_n_ > _chunk_size) {                      \
-            vml_func((MKL_INT) _chunk_size, in1p, op1p); \
-            _n_ -= _chunk_size;                          \
-            in1p += _chunk_size;                         \
-            op1p += _chunk_size;                         \
-        }                                                \
-        if (_n_) {                                       \
-            vml_func((MKL_INT) _n_, in1p, op1p);         \
-        }                                                \
+#define CHUNKED_VML_CALL2(vml_func, n, type, in1, op1)                          \
+    do {                                                                        \
+        npy_intp _n_ = (n);                                                     \
+        const npy_intp _chunk_size = MKL_INT_MAX;                               \
+        type *in1p = (type *) (in1);                                            \
+        type *op1p = (type *) (op1);                                            \
+        while (_n_ > 0) {                                                       \
+            npy_intp _current_chunk = (_n_ > _chunk_size) ? _chunk_size : _n_;  \
```

Comment:
Would it make sense to use `std::min(_n_, _chunk_size)` instead?

Reply:
It's C file, so there's no built-in `min` or `max`